applications/asset_tracker_v2/doc/sensor_module.rst

```diff
@@ -60,12 +60,11 @@ The threshold is set in one of the following two ways:
 
 Both events contain an accelerometer threshold value ``accelerometer_threshold`` in m/s2, present in the event structure.
 
-Motion detection in the module is enabled and disabled in turn to control the number of :c:enum:`SENSOR_EVT_MOVEMENT_DATA_READY` events that are sent out by the sensor module.
-This functionality acts as flow control.
-It prevents the sensor module from sending events to the rest of the application if the device is accelerating over the threshold for extended periods of time.
+Motion detection is enabled and disabled according to the device mode parameter, received in the aforementioned configuration events.
+Motion detection will be enabled in passive mode and disabled in active mode.
 
-The application module controls this behavior through the :c:enum:`APP_EVT_ACTIVITY_DETECTION_ENABLE` and :c:enum:`APP_EVT_ACTIVITY_DETECTION_DISABLE` events.
-The sensor module will only send out a :c:enum:`SENSOR_EVT_MOVEMENT_DATA_READY` event if it detects movement and activity detection is enabled.
+The sensor module will send out a :c:enum:`SENSOR_EVT_MOVEMENT_ACTIVITY_DETECTED` event if it detects movement.
+Similarly, :c:enum:`SENSOR_EVT_MOVEMENT_INACTIVITY_DETECTED` is sent out if there was no movement for a while.
```

Comment:
```suggestion
The sensor module sends out a :c:enum:`SENSOR_EVT_MOVEMENT_ACTIVITY_DETECTED` event if it detects movement.
Similarly, :c:enum:`SENSOR_EVT_MOVEMENT_INACTIVITY_DETECTED` is sent out if there is no movement in a while.
```
Should we be more specific with the time? How long is a while?

Reply:
This will be made more specific in an upcoming PR when we enable users to configure this at runtime.